paimon-core/src/main/java/org/apache/paimon/table/Table.java

```diff
@@ -58,6 +59,9 @@ public interface Table extends Serializable {
     /** Optional comment of this table. */
     Optional<String> comment();
 
+    /** Optional statistics of this table. */
+    Optional<Stats> statistics();
```

Comment:
add `@Experimental`

and add `Experimental` to Stats, ColStats. they are public api

Reply:
`Stats` can be `Statistics`